WordPress/Classes/Services/AccountService.h

```diff
@@ -4,6 +4,7 @@
 @class WPAccount, Blog;
 
 extern NSString *const WPAccountDefaultWordPressComAccountChangedNotification;
+extern NSString *const WPAccountWordPressComAccountEmailAndDefaultBlogUpdatedNotification;
```

Comment:
Zomg, 66 chars. Could this const be smaller? Please? Pretty please with cherry on top?